### ISSUE TITLE
Increased serum calprotectin level 

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10319]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
PMID: 27895095 **Calprotectin** is an acute-phase protein produced by monocytes and neutrophils in the circulation and inflamed tissues. Calprotectin seems to be more sensitive than CRP, being able to detect minimal residual inflammation and is a candidate biomarker in inflammatory diseases. High serum levels are associated with some severe manifestations of rheumatic diseases, such as glomerulonephritis and lung fibrosis. Calprotectin levels in other fluids, such as saliva and synovial fluid, might be helpful in the
diagnosis of rheumatic diseases

PMID: 36344353 Osteopontin is a glycoprotein that takes part in several inflammatory pathways and is highly expressed in tissue biopsies from patients with GCA (Giant cell arteritis) . Serum levels of this biomarker are significantly higher in  patients with active disease and are not modified by tocilizumab.  Likewise, **serum calprotectin** correlates with disease activity in patients  with GCA undergoing treatment with glucocorticoids, and may possibly maintain such a predictive role also during IL-6 blocking therapy.

With the growing use of Tocilizumab in GCA, hindering the relevance of traditional biomarkers such as CRP or ESR, the need for a new
biomarker becomes urgent to assist clinicians in the evaluation of GCA patients. In this study, we showed that calprotectin may be of use to fulfill this role, as a promising tool to detect GCA activity.

**Parent term (use hpo.jax.org/app)**
Acute phase response HP:0033331

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Giant cell arteritis ORPHA:397
ORPHA:92  Juvenile idiopathic arthritis
ORPHA:829 Adult-onset Still disease

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6000502
- label: Elevated circulating calprotectin concentration
- parents: [Abnormal circulating protein concentration (HP:0010876)](https://hpo.jax.org/app/browse/term/HP:0010876)
- pmid: PMID:27895095
- def: The concentration of calprotectin in the blood circulation is above the upper limit of normal.
- comment: Calprotectinis an acute-phase protein produced by monocytes and neutrophils in the circulation and inflamed tissues.
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10319
- synonyms:
